src/app/portlets/dot-edit-page/shared/services/dot-edit-layout-guard/dot-edit-layout-guard.service.ts

```diff
@@ -1,16 +1,14 @@
-import { DotRouterService } from './../../../../../api/services/dot-router/dot-router.service';
 import { Injectable } from '@angular/core';
 import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
-import { Observable } from 'rxjs/Observable';
 
 /**
  * Route Guard that checks if a User is logged in.
  */
 @Injectable()
 export class DotEditLayoutGuardService implements CanActivate {
-    constructor(private dotRouterService: DotRouterService) {}
+    constructor() {}
 
-    canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): boolean {
+    canActivate(route: ActivatedRouteSnapshot, _state: RouterStateSnapshot): boolean {
```

Comment:
New guys for ```/layout``` if the user try to get in from url and have no permissions the resolver catch it, if the user try to go here from ```/content``` and have no access it will return false because ```route.parent.data.content.page.canEdit```